### PR TITLE
Allowed creation of Experience entity in database with proper

### PR DESCRIPTION
**Pull request by GabrielRodriguez66**
field validators. Implemented the corresponding form for experience
registration, added experience registration route and added the option
in the nav bar.

### CODE REVIEW

File: .idea/ResumeBuilder_.iml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Joshuacruzc) [inline .idea/ResumeBuilder_.iml:1]: Add .idea/ directory to .gitignore file.

File: resumebuilder/routes.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+from resumebuilder.models import User, Experience, Tag
 
 
-@app.route('/', methods=['GET', 'POST'])
+@app.route('/index', methods=['GET', 'POST'])
 def index():
     return render_template('index.html')
 
-@app.route('/register', methods=['GET', 'POST'])
+@app.route('/', methods=['GET', 'POST'])
```
Comment (Joshuacruzc) [inline resumebuilder/routes.py]: revert

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (GabrielRodriguez66) [inline .gitignore]: Añadí el directorio de .idea/

File: resumebuilder/routes.py [hunk 4]
```diff
@@ -9,7 +10,7 @@
 def index():
     return render_template('index.html')
 
-@app.route('/', methods=['GET', 'POST'])
+@app.route('/register', methods=['GET', 'POST'])
```
Comment (GabrielRodriguez66) [inline resumebuilder/routes.py]: Revertí el cambio erróneo

File: .gitignore (hunk 3 above)
Comment (Joshuacruzc) [reply .gitignore]: slash is not necessary.  
place comments in Discussions and close them if you did what was asked for.

File: resumebuilder/templates/layout.html [hunk 5]
```diff
@@ -23,9 +23,12 @@
   <!-- Links -->
 <div id="collapsibleNavbar" class="collapse navbar-collapse">
   <ul class="navbar-nav ml-auto">
-      {%  if current_user.is_authenticated %}
+      {% if current_user.is_authenticated %}
+          <li>
```
Comment (Joshuacruzc) [inline resumebuilder/templates/layout.html]: give <li> tag class of nav-item and <a> tag class of nav-link for consistency and css styling.

File: resumebuilder/routes.py [hunk 6]
```diff
@@ -1,11 +1,12 @@
...
+
 from resumebuilder import app, db, bcrypt
-from resumebuilder.forms import RegistrationForm, LoginForm
-from resumebuilder.models import User
+from resumebuilder.forms import RegistrationForm, LoginForm, ExperienceForm
+from resumebuilder.models import User, Experience, Tag
 
 
-@app.route('/', methods=['GET', 'POST'])
+@app.route('/index', methods=['GET', 'POST'])
```
Comment (Joshuacruzc) [inline resumebuilder/routes.py]: revert

File: resumebuilder/routes.py [hunk 7]
```diff
@@ -43,4 +44,31 @@ def login():
 @app.route('/logout')
 def logout():
     logout_user()
-    return redirect(url_for('index'))
\ No newline at end of file
+    return redirect(url_for('index'))
+
+
```
Comment (Joshuacruzc) [inline resumebuilder/routes.py:50]: include @login_required decorator from flask_login to prevent page being accessed by unauthenticated users.

**Pull request merged**